src/Features/LanguageServer/Protocol/ProtocolConstants.cs

```diff
@@ -11,5 +11,7 @@ internal class ProtocolConstants
         public const string RazorCSharp = "RazorCSharp";
 
         public static ImmutableArray<string> RoslynLspLanguages = ImmutableArray.Create(LanguageNames.CSharp, LanguageNames.VisualBasic, LanguageNames.FSharp);
+
+        public const string RoslynLspLanguagesContract = "RoslynLspLanguages";
```

Comment:
This might deserve a bit of a comment here for what this is.

Reply:
Or maybe something like RoslynLspLanguageHandlerContract makes it a bit clearer this applies to the handlers? Otherwise this could be a string for...anything?

Reply:
updated in https://github.com/dotnet/roslyn/pull/59600